vortex-btrblocks/src/integer.rs

```diff
@@ -241,8 +241,11 @@ impl Scheme for FORScheme {
 
         // Difference between max and min
         let full_width: u32 = stats.src.ptype().bit_width().try_into().vortex_unwrap();
-        let padding = 64 - full_width;
-        let bw = full_width + padding - stats.typed.max_minus_min().leading_zeros();
+        let bw = match stats.typed.max_minus_min().checked_ilog2() {
+            Some(l) => l + 1,
+            // If max-min == 0, it we should use a different compression scheme.
+            None => return Ok(0.0),
+        };
```

Comment:
I think this is simpler and doesn't have any hard-coded values

Reply:
can you then delete the check for `bw == 0`

Reply:
done